cni-plugin/Makefile

```diff
@@ -140,13 +141,31 @@ $(DEPLOY_CONTAINER_FIPS_MARKER): Dockerfile build fetch-cni-bins
 	$(MAKE) retag-build-images-with-registries VALIDARCHES=$(ARCH) IMAGETAG=latest-fips LATEST_IMAGE_TAG=latest-fips
 	touch $@
 
+CN_FLAGS=-ldflags "-X github.com/containernetworking/plugins/pkg/utils/buildversion.BuildVersion=$(GIT_VERSION)"
+
+CONTAINERNETWORKING_PLUGINS_CREATED=.containernetworking-plugins-$(ARCH).created
+CONTAINERNETWORKING_PLUGINS_CLONED=.containernetworking-plugins.cloned
+
+$(CONTAINERNETWORKING_PLUGINS_CLONED):
```

Comment:
I think this will get stuck on an old branch when you pull an update to the calico repo that changes CNI_VERSION

I've used this pattern in the past to deal with this:
```
TOUCHFILE=.foo-bar-cloned.$(VERSION)

$(TOUCHFILE):
    rm -rf <the repo>  .foo-bar-cloned.*
    ... clone it ...
    touch $@
```